pkg/azureclients/privatednszonegroupclient/interface.go

```diff
@@ -20,15 +20,25 @@ import (
 	"context"
 
 	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2021-02-01/network"
+
+	"sigs.k8s.io/cloud-provider-azure/pkg/retry"
+)
+
+const (
+	// APIVersion is the API version.
+	APIVersion = "2021-02-01"
+	// AzureStackCloudAPIVersion is the API version for Azure Stack
+	AzureStackCloudAPIVersion = "2016-04-01"
```

Comment:
which version should we use here?

Reply:
Offline discussed. Will remove azure stack related code because it is not supported yet.